linera-service/src/linera.rs

```diff
@@ -1005,6 +1005,10 @@ enum NetCommand {
         /// The number of shards per validator in the local test network. Default is 1.
         #[structopt(long, default_value = "1")]
         shards: usize,
+
+        /// The table name used
```

Comment:
Maybe: "The name for the database table to store the chain data in."

Reply:
Ok.